mtest/CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ if (MSVC)
       set(_all_h_file "${PROJECT_SOURCE_DIR}/all.h")
 endif (MSVC)
 
-if (Qt5Widgets_VERSION VERSION_GREATER_EQUAL "5.11.0")
+if (Qt5Widgets_VERSION VERSION_GREATER_EQUAL "5.99.0")
```

Comment:
Huh?

Reply:
@mattmcclinch recommended this fix for a build failure that I had with the mtests.

Reply:
This was just a hack to get mtests to compile locally. It was not supposed to be part of the pull request. For some reason, qtquick_compiler_add_resources is not doing the job, so this forces it to fall back to using QT5_ADD_RESOURCES instead. Interestingly enough, however, qtquick_compiler_add_resources seems to work just fine when called from mscore/CMakeLists.txt.

Reply:
Yes, of course. I don't plan to include this in the completed pull request. :^)

Reply:
It is though, now?

Reply:
IIRC this has been resolved differently meanwhile?

Reply:
Ah, I see, in #6477, not yet merged. Maybe you cherry pick that one meanwhile?